lib/go/Makefile

```diff
@@ -76,10 +76,11 @@ export PATH := $(shell pwd):$(PATH)
 ########################################################################
 ##                              BUILD                                 ##
 ########################################################################
-CSI_GO := csi/csi.pb.go
-CSI_A := csi.a
 CSI_PROTO := ../../csi.proto
```

Comment:
Hi @jdef,

If the language binding's path/package structure is going to be versioned then shouldn't `csi.proto` as well?

Reply:
I believe this is necessary for C++ to version the `.proto` file.